sys-cluster/easybuild-easyblocks/easybuild-easyblocks-4.9.0.ebuild

```diff
@@ -0,0 +1,37 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
+DISTUTILS_USE_PEP517=setuptools
+PYPI_NO_NORMALIZE=1
+
+inherit distutils-r1
+
+SUB_P=$(ver_cut 1)
+
+DESCRIPTION="Provides a collection of easyblocks for EasyBuild."
+HOMEPAGE="
+	https://easybuild.io/
+	https://github.com/easybuilders/easybuild-easyblocks
+"
+
+LICENSE="GPL-2"
+SLOT="0"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/easybuilders/easybuild-easyblocks"
+	SUB_P=9999
```

Comment:
```suggestion
```

This I think is unnecessary `ver_cut 1` should already return 9999 when that is the PV.